### DISCUSSION TITLE
Welcome to autowx2 Discussions!

### DISCUSSION THREAD

**Discussion by filipsPL** [filipsPL/autowx2#104] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thecivvie** [comment]
Great. I have a few questions and the issues tab isn't really the right place.
I sorted my startup by rereading through all the comments.
